frontend/docs/admin-guide/getting-started.md

```diff
@@ -265,6 +265,6 @@ Listed below are security best practices and design principles implemented by th
 
 ## Dynamic IAM Roles
 
-When users log into their SageMaker notebook instances, permissions to AWS resources and APIs are managed by the execution role associated with the instance. The creation of AWS Identity & Access Management (IAM) roles within an account is typically managed by a separate organizational process that requires manual review. Since the {{ $params.APPLICATION_NAME }} application’s goal is to provide users a frictionless mechanism to self-organize and create projects for collaboration, the turnaround time for role creation is a limiting factor. The use of coarse-grained policies to reduce the number of roles required to manage the {{ $params.APPLICATION_NAME }} environment limits the ability to isolate project resource access to only the authorized {{ $params.APPLICATION_NAME }} users when requests originate from a SageMaker notebook instance. IdP credentials cannot be stored directly on the notebook instances for a number of reasons including security concerns. The “identity” of the notebook becomes obfuscated if a common IAM role is implemented across the user community. Additionally, users require integration with external systems to access data, code repositories, and software libraries relevant for their ML use cases which further increases the need for more granular access controls.
+When users log into their SageMaker notebook instances, permissions to AWS resources and APIs are managed by the execution role associated with the instance. The creation of AWS Identity & Access Management (IAM) roles within an account is typically managed by a separate organizational process that requires manual review. Since the {{ $params.APPLICATION_NAME }} application’s goal is to provide users with a frictionless mechanism to self-organize and create projects for collaboration, the turnaround time for role creation is a limiting factor. The use of coarse-grained policies to reduce the number of roles required to manage the {{ $params.APPLICATION_NAME }} environment limits the ability to isolate project resource access to only the authorized {{ $params.APPLICATION_NAME }} users when requests originate from a SageMaker notebook instance. IdP credentials cannot be stored directly on the notebook instances for a number of reasons, including security concerns. The “identity” of the notebook becomes obfuscated if a common IAM role is implemented across the user community. Additionally, users require integration with external systems to access data, code repositories, and software libraries relevant to their ML use cases, which further increases the need for more granular access controls.
```

Comment:
Hard to tell what changed in this line and the one below via github. Can you clarify what changed?

Reply:
Yeah I am not sure why it isn't showing the changes…

This line…
- `provide users a` -> `provide users with a`
- `relevant to their ML use cases which further increases` -> `relevant to their ML use cases, which further increases`

One below…
- `attaches a standardized set of IAM policies` -> `attach a standardized set of IAM policies`
- a couple `MLSpace` -> `{{ $params.APPLICATION_NAME }}`